use_with_openvino.md

```diff
@@ -0,0 +1,84 @@
+# Use vLLM with OpenVINO
+
+## Build Docker Image
+
+```bash
+git clone --branch openvino-model-executor https://github.com/ilya-lavrenov/vllm.git
+cd vllm
+docker build -t vllm:openvino -f Dockerfile.openvino .
+```
+
+Once it successfully finishes you will have a `vllm:openvino` image. It can directly spawn a serving container with OpenAI API endpoint or you can work with it interactively via bash shell.
+
+## Use vLLM serving with OpenAI API
+
+_All below steps assume you are in `vllm` root directory._
+
+### Start The Server:
+
+```bash
+# It's advised to mount host HuggingFace cache to reuse downloaded models between the runs.
+docker run --rm -p 8000:8000 -v $HOME/.cache/huggingface:/root/.cache/huggingface vllm:openvino --model meta-llama/Llama-2-7b-hf --port 8000 --disable-log-requests --swap-space 50
+
+### Additional server start up parameters that could be useful:
+# --max-num-seqs <max number of sequences per iteration> (default: 256)
+# --swap-space <GiB for KV cache> (default: 4)
+```
+
+### Request Completion With Curl:
+
+```bash
+curl http://localhost:8000/v1/completions -H "Content-Type: application/json" -d '{"model":"meta-llama/Llama-2-7b-hf", "prompt": "What is the key advantage of Openvino framework","max_tokens": 300, "temperature": 0.7}'
+```
+
+### Run Benchmark
+
+Let's run [benchmark_serving.py](https://github.com/ilya-lavrenov/vllm/blob/openvino-model-executor/benchmarks/benchmark_serving.py):
+
+```bash
+cd benchmarks
+# Download dataset
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
+
+# Install PyPI dependencies (vLLM and aiohttp seem to be enough)
+pip3 install vllm aiohttp
```

Comment:
why do we need to install `vllm`? will it install from web?

Reply:
Benchmark script has some dependencies and installing vllm seems to be the easiest way to get those dependencies to the environment on the client side.